Error when generating Spring from 1.5.0 RC

There is a workaround to suppress the validation, noting down for possible fix before 1.5.0 release.

```
PS C:\Users\jspetrak\Workspace\Bileto\osdm-5> .\gradlew.bat openApiGenerate
> Task :openApiGenerate FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':openApiGenerate'.
> There were issues with the specification. The option can be disabled via validateSpec (Maven/Gradle) or --skip-validate-spec (CLI).
   | Error count: 1, Warning count: 1
  Errors:
        -attribute paths.'/coachLayouts'(get).parameters.There are duplicate parameter values
  Warnings:
        -attribute paths.'/coachLayouts'(get).parameters.There are duplicate parameter values
```

--- a/specification/v1.5.0-rc/OSDM-online-api-v1.5.0-rc.yml
+++ b/specification/v1.5.0-rc/OSDM-online-api-v1.5.0-rc.yml
@@ -2631,12 +2631,6 @@ paths:
         - $ref: "#/components/parameters/acceptLanguage"
         - $ref: "#/components/parameters/traceParent"
         - $ref: "#/components/parameters/traceState"
-        - in: query
-          name: page
-          description: >-
-            can be used for pagination control
-          schema:
-            type: string
       responses:
         "200":
           description: coach layouts

